lib/fasterer/file_traverser.rb

```diff
@@ -76,7 +76,7 @@ def all_files
     end
 
     def root_dir
-      @root_dir ||= Pathname('.')
+      @root_dir ||= Pathname(".")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.